doc/usage/builders/html.rst

```diff
@@ -1,17 +1,194 @@
-.. highlight:: python
+=============
+HTML Builders
+=============
 
-.. _html-themes:
+All HTML builders generate HTML output files but vary in how this output is
+structured and what additional artifacts are included. Specifically, the
+`Directory HTML builder`_ and `Single HTML builder`_ are variants of the
+default `Standalone HTML builder`_ that generate output in different formats,
+while the remaining builders generate the same output but also generate
+additional files required by specific applications or use cases.
+
+
+Standalone HTML builder
+-----------------------
+
+This is the standard HTML builder.  Its output is a directory with HTML files,
+complete with style sheets and optionally the reST sources.  There are quite a
+few configuration values that customize the output of this builder, see the
+chapter :ref:`html-options` for details.
+
+.. module:: sphinx.builders.html
+.. class:: StandaloneHTMLBuilder
+
+   .. autoattribute:: name
+
+   .. autoattribute:: format
+
+   .. autoattribute:: supported_image_types
```

Comment:
I wonder this block is necessary for users. I feel the descriptions you wrote are enough good.
The name is only an important field. But we don't need to use autodoc to document it.
But we already have described it in the command reference of sphinx-build:
https://www.sphinx-doc.org/en/3.x/man/sphinx-build.html?highlight=sphinx-build#cmdoption-sphinx-build-b

Reply:
I think the `supported_image_types` field is useful, given that not all output formats support all image types. As for `name`, while we could document this manually, it seems simpler to automatically document this?

Reply:
I understand about `supported_image_types`. How about docstrings of builder classes? I think it is duplicated with the descriptions you wrote.

About `name`, it will never (or very very rarely) be changed in the future. So automation does not help us.